racket/src/racket/cmdline.inc

```diff
@@ -755,6 +755,10 @@ static Scheme_Object *get_log_level(char *prog, char *real_switch, const char *e
   l =  scheme_make_null();
 
   while (1) {
+    while (*str && *str == ' ') {
```

Comment:
Probably, but `' '` is hardcoded elsewhere in the same routine and I don't want to get into heavily refactoring it. That can be a separate issue.